pypeman/channels.py

```diff
@@ -480,6 +495,51 @@ def graph_dot(self, end=''):
             for entry in sub.graph_dot(end=end):
                 yield entry
 
+    def _init_callback_nodes(self, *callback_nodes):
+        callback_nodes = [node for node in flatten(callback_nodes) if node]
+        for node in callback_nodes:
+            node.channel = self
```

Comment:
How can a function that is called init_callback_nodes() change channels of existing nodes?